README.md

```diff
@@ -1138,12 +1138,12 @@ class BankAccount
         $this->balance -= $amount;
     }
 
-    public function depositBalance(int $amount): void
+    public function deposit(int $amount): void
     {
         $this->balance += $amount;
     }
 
-    public function getBalance(): int
+    public function balance(): int
```

Comment:
This sounds like operation, since it's a verb. 
If `deposit()` woudl have same meaning, it should have return `$deposit`.

I'd expect getter for anything would to have `get<Anything>()` name.

Reply:
About getters see #31

Reply:
tl;dr; + I don't get it

Reply:
I thank Peter for organising this PR off the back of my initial comment, however I would also not have renamed 'getBalance'. That function is clearly different from the other two, and should keep the 'get' prefix IMHO. I don't think that is inconsistent; if instead of withdraw/deposit we had one function called setBalance(), we would keep the 'set' prefix.

'withdraw' is an action
'deposit' is an action
'set balance' is an action
'get balance is an action

Reply:
@someonefamous use of prefixes `get` and `set` is [antipattern](https://martinfowler.com/bliki/AnemicDomainModel.html).
This has already been discussed in #31.